Winapp2.ini

```diff
@@ -57,6 +57,16 @@ RegKey1=HKCU\Software\Chromium\BrowserExitCodes
 RegKey2=HKCU\Software\CocCoc\Browser\BrowserExitCodes
 RegKey3=HKCU\Software\Google\Chrome\BrowserExitCodes
 
+[Chocolatey *]
+LangSecRef/Section=3024
+Detect/DetectFile=%CommonAppData%\chocolatey
+Default=False
+FileKey1=%CommonAppData%\chocolatey\logs\choco.summary.log
+FileKey2=%CommonAppData%\chocolatey\logs\chocolatey.log
+FileKey3=%CommonAppData%\chocolatey\lib\*\*.txt
+FileKey4=%LocalAppData%\Temp\chocolatey\|*.*|REMOVESELF
```

Comment:
Keys needn't include %LocalAppData%\Temp\.. keys as it's presumed that whichever cleaner is running this entry will empty the temp folder anyway